fw/preamp/src/port_defs.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * AmpliPi Home Audio
+ * Copyright (C) 2021 MicroNova LLC
+ *
+ * Port definitions
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "port_defs.h"
+
```

Comment:
Add comment:
// Enable pin mapping for each channel's 4 sources
// Each channel can enable all or none of its four sources. This firmware currently only allows one to be enabled at time.